src/crudtask.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-alert */
+// eslint-disable-next-line import/no-named-default
```

Comment:
- [ ] please avoid disabling eslint rules as this rules enforce you to write better code hence making you a better developer